site/content/docs/5.2/components/pagination.md

```diff
@@ -121,12 +121,72 @@ Or with `.justify-content-end`:
 <!-- Boosted mod -->
 ## Responsive behavior
 
-To avoid pagination wrapping or overflowing, we limit the number or pagination item to display depending on breakpoint.
+{{< callout warning >}}
+**Heads up—our responsive pagination is experimental and opt-in as of v5.2.1!** We included it in our documentation's CSS to demonstrate it for you, but it's disabled by default. Keep reading to learn how to enable it in your projects.
+{{< /callout >}}
+
+By default, every item is visible. Here is an example **without** our responsive behavior.
```

Comment:
This is confusing. Both our solutions are in fact responsive. The second (opt-in) just offers a smart version of the responsive behavior.

When it is said before "Heads up—our responsive pagination is experimental and opt-in as of v5.2.1!". That's the same think, the smart responsive behavior is experimental, not the normal one.